public/index.html

```diff
@@ -8,8 +8,79 @@
 </head>
 <body>
     <div class="wrap">
-        <h1 class="test-headline">Slidy</h1>
-        <p class="test-paragraph">Залиш свій слід у світі подорожей</p>
-    </div>
+        <div class="container">
```

Comment:
Bad name for a class. Classes should not be named "container" or "block" or "text" etc. They must be named using BEM methodology in depends of which role of them on website

Reply:
i deleted this div